core/src/main/java/nl/nn/adapterframework/doc/XsdVersion.java

```diff
@@ -0,0 +1,22 @@
+package nl.nn.adapterframework.doc;
```

Comment:
License info

Reply:
Gedaan. Maar ik heb niet gekeken of ik "2020, 2021" of alleen "2021" moet neerzetten.